app/models/user.rb

```diff
@@ -11,4 +11,26 @@ class User < ApplicationRecord
 
   enum active_status: {active: 0, inactive: 1}
   enum donor_status: {donor: 0, recipient: 1}
+  before_save :normalize_phone
```

Comment:
⛏️ Consider using `before_validations` instead, so phone numbers are normalized before `.valid?` is evaluated.